code/modules/vehicles/wheelchair.dm

```diff
@@ -138,6 +138,14 @@
 /obj/item/wheelchair/attack_self(mob/user)  //Deploys wheelchair on in-hand use
 	deploy_wheelchair(user, user.loc)
 
+/obj/item/wheelchair/afterattack(obj/target, mob/user, proximity)
+	. = ..()
```

Comment:
Should use `interact_with_atom` now if possible

Reply:
updated that plus rollerbeds and bodybags since those came to mind as having the same behavior

Reply:
Now you can add code improvement to the changelog lol